scripts/buildPages.sh

```diff
@@ -3,7 +3,11 @@ for i in "${array[@]}"
 do
   echo "Building $i docs ..."
 	npm run docs:$i
-  mkdir --parents ./pages/docs/$i
+  if [[ "$OSTYPE" == "darwin"* ]]; then
+    mkdir -p ./pages/docs/$i
+  else
+    mkdir --parents ./pages/docs/$i
+  fi
```

Comment:
```suggestion
  mkdir -p ./pages/docs/$i
```
We can drop the OS check, `-p` also works on git bash on Windows.